code/modules/antagonists/wizard/abilities/animal.dm

```diff
@@ -115,14 +115,18 @@ var/list/animal_spell_critter_paths = list(/mob/living/critter/small_animal/cat,
 		smoke.attach(target)
 		smoke.start()
 
-		if (target.mind && (target.mind.assigned_role != "Animal") || (!target.mind || !target.client))
+		if (target.mind || (!target.mind || !target.client))
```

Comment:
```suggestion
		if (!target.mind || !target.client)
```
Currently checking mind || !mind which always returns TRUE

Also lets you polymorph ghost critters, etc., if changes are kept

Reply:
onStart() and onUpdate() for the action bar contain ishuman() checks. 
cast() for the targetable spell contain ishuman() checks.

Perhaps we should have another check onEnd() but no non human should even be making it to this point so the assigned_role check would be meaningless.